### PR TITLE
GH#28646: added port to TCP protocol table

### PR DESCRIPTION
**Pull request by skrthomas**
GitHub issue burndown, #27554 
Direct link to doc: https://deploy-preview-30823--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/installing-bare-metal.html#installation-network-user-infra_installing-bare-metal

### CODE REVIEW

File: modules/installation-network-user-infra.adoc [hunk 1]
```diff
@@ -101,6 +101,8 @@ the Cluster Version Operator on port `9099`.
 |`10256`
 |openshift-sdn
 
+|`1936`
+|Metrics
```
Comment (bobfuru) [inline modules/installation-network-user-infra.adoc:105]: @sagidlow - I suggest making this the first cell in the `TCP` entry based on numerical ordering. WDYT?

**Review by bobfuru — COMMENTED** [review]
A question about ordering. Also, I'm thinking this should be CP'ed to 4.5+ so I added those labels. But if that's incorrect, let me know. Thanks!

File: modules/installation-network-user-infra.adoc (hunk 1 above)
Comment (skrthomas) [reply modules/installation-network-user-infra.adoc:105]: Sounds reasonable to me!

Comment (skrthomas) [reply modules/installation-network-user-infra.adoc]: @bobfuru Done :)

**Comment by bobfuru** [comment]
LGTM!

**Pull request merged**